pkg/csi/plugins/nodeserver.go

```diff
@@ -577,3 +592,38 @@ func (ns *nodeServer) prepareSessMgr(workDir string) error {
 
 	return nil
 }
+
+func createSymlink(targetPath, mountPath string) error {
```

Comment:
Please move this function to utils pkg and create unit test for this change.

Reply:
done.